trk/src/Screens/TabScreens/Record/Backend/logic.js

```diff
@@ -125,11 +163,14 @@ export const useHomeScreenLogic = (props) => {
                 </>
             );
         } else {
+
+            const messageComponent = selectedMessage.split(/(?<=[.,!?  ])\s/).map((sentence, index) => (
+                <Text key={index} style={index === 0 ? styles.tapText : styles.sentence}>{sentence}</Text>
+            ));
+
```

Comment:
the message is split into multiple sentences if there is a punctuation mark (.,!?) or a double space. Each sentence is maped into a <Text></Text> component. The first text component has styles.tapText and all remaining have style.sentence (this is because the first sentence needs more marginTop than the other sentences)